Reference/using-ioc.md

```diff
@@ -141,3 +141,103 @@ If you don't register assemblies that contain controllers you may end up with YS
 ##Things to note
 
 We use a custom MVC controller builder in our code called `Umbraco.Web.Mvc.MasterControllerFactory`, which needs to always be the default controller factory, if you change this Umbraco will probably not work anymore. The good news is that you can specify 'slave' factories so you can specify custom controller factories for different purposes. You would just need to create a new class that inherits from `Umbraco.Web.Mvc.IFilteredControllerFactory` and ensure that the class is public (so it can be found). If your IoC implementation affects the default controller factory, you may have to modify it in order to support this implementation. For the most part, most IoC frameworks will just target setting a custom DependencyResolver which is 100% ok.
+
+##Unity Example
+
+Install the Unity, Unity.Mvc and Unity.AspNet.WebApi packages.
+Remove the Mvc and Api specific files created, we will be merging those files shortly.
+Leave the UnityConfig file as is.
+
+Now create the following files to configure Unity correctly for an Umbraco site.
+
+	class UnityEvents : IApplicationEventHandler
+	{
+	    public void OnApplicationStarted(
+		UmbracoApplicationBase httpApplication,
+		ApplicationContext applicationContext
+	    )
+	    {
+		var container = UnityConfig.GetConfiguredContainer();
+
+		// If you intend to use DI inside filter attributes
+		FilterProviders.Providers.Remove(FilterProviders.Providers.OfType<FilterAttributeFilterProvider>().First());
+		FilterProviders.Providers.Add(new UnityFilterAttributeFilterProvider(container));
```

Comment:
It works with my umbraco installation but might be better left out of the docs. will remove.